### ISSUE TITLE
Incorrect method description in README.md

### ISSUE THREAD

**Issue by grahamharper** [moust/cordova-plugin-videoplayer#9]
The method description on line 21 in README.md uses `.get()` is this correct? It doesn't look like this method exists. Shouldn't it be `.play()`?

```
VideoPlayer.get(path, [options], [completeCallback], [errorCallback]);
```

**Comment by moust** [comment]
You're right, it should be `play()` method. I will fix it with my next commit. 
Thanks.

**Comment by moust** [comment]
Fixed (029b624).
